### ISSUE TITLE
Device ID validation failed

### ISSUE THREAD

**Issue by dlang9159** [syncthing/syncthing#2692]
I'm using syncthing v0.12.12 on to devices. I wanted to connect these, but the Device-ID I copied generates an error on the other computer and vice versa. The computers I conntected in previous versions seem to work fine.

Update: Adding to another computer worked without problem.
![screenshot 32](https://cloud.githubusercontent.com/assets/1560835/12288168/0dadde52-b9d2-11e5-8881-6b458c294648.jpg)

**Comment by AudriusButkevicius** [comment]
Can you upload the certificates somewhere, and just delete them and restart syncthing to get new ones for yourself? Also are you sure you copied it without modifying?

**Comment by dlang9159** [comment]
After restarting both syncthing nodes and copying the device ID directly out of the webinterface it finally worked. 
Previously I copied the ID out of an email text.
Just as a test, I copied both IDs in an editor, they are both the same text (although only one worked when copied from the editor back into the add device dialog)
I changed the editors coding from "UTF-8 with BOM" to "ANSI and "â€‹" was appended after the not working device ID.
